scripts/gufi_config.py

```diff
@@ -67,9 +67,16 @@
 
 import gufi_common
 
-# default configuration file location
+# configuration file location
 DEFAULT_PATH = '/etc/GUFI/config'
 
+# helper function to allow configurable paths to the gufi config file
+def config_path():
+  try:
+    return os.environ['GUFI_CONFIG']
+  except KeyError:
+    return DEFAULT_PATH
```

Comment:
Why change it from just checking if the key is in `os.environ`? The cost of exceptions is basically 0 when no exception is raised, and I expect that both will occur relatively frequently. While this shouldn't be looped, why even have this issue to begin with?

Reply:
I'll change it back. The only reason I preferred this approach was because I couldn't figure out how to avoid either creating a new variable or hard-coding the string in two places.
So it has to be either:
```
gufi_config_env = 'GUFI_CONFIG' # create a variable
if gufi_config_env in os.environ:
  return os.environ[gufi_config_env]
```
or
```
if 'GUFI_CONFIG' in os.environ:
  return os.environ['GUFI_CONFIG'] # use the hard-code string literal a second time
```
So, to my sense of aesthetics, I thought it was clean to use try/except which avoids creating a new variable or using the same string literal twice.

You prefer the creating a variable way? LMK please and I'll do whatever.

Reply:
I prefer the first way

Reply:
Reverted back to the first way.